src/main/java/org/apache/commons/text/similarity/LevenshteinResults.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.text.similarity;
+
+/**
+ * Container class to store Levenshtein distance between two character sequence.
+ * It also stores the count of inserts, deletions and substitutions needed to
+ * change one character sequence to other.
+ * 
+ */
+public class LevenshteinResults {
```

Comment:
Hi @drajakumar 

Last comment in the pull request before merging I think :-)

What do you think about adding #toString(), #hashcode() and #equals() methods to this class? If you don't have time right now, we can merge and do that later.

Other than that, happy to merge :clap: